stix_shifter_modules/nozomi/stix_translation/transformers.py

```diff
@@ -0,0 +1,135 @@
+        except ValueError:
+            LOGGER.error("%s connector error, cannot return obj value %s with default param 'kill_chain_name'",
+                         connector, obj)
+        return None
+
+
+class ValidateMacAddr(ValueTransformer):
+    """Regex to check valid MAC address
+    Example:
+        Input: Invalid mac address
+        Output: None
+    """
+    @staticmethod
+    def transform(obj):
+        try:
+            pattern = re.compile(r'^(([0-9a-fA-F]{2}[:-]){5}([0-9a-fA-F]{2})|([0-9a-fA-F]{3}[\.]){3}([0-9a-fA-F]{3}))$')
+            if re.search(pattern, str(obj)):
+                return obj
+            return None
+        except ValueError:
+            LOGGER.error("%s connector error, cannot validate mac address: %s", connector, obj)
+            raise
+
+
+class ToFindingType(ValueTransformer):
+    """A value transformer for expected finding_type value
+    Example:
+        Input: empty string or other than empty string
+        Output: alert or threat
+    """
+    @staticmethod
+    def transform(obj):
+        try:
+            if obj in ("", "alert"):
+                return "alert"
+            return "threat"
+        except ValueError:
+            LOGGER.error("%s connector error, cannot convert input to finding type : %s", connector, obj)
+            raise
+
+
+class ToSeverityValue(ValueTransformer):
+    """A value transformer for converting severity value
+    Example:
+        Input: 9.0 or 9 or '9.0'
+        Output: 90
+    """
+    @staticmethod
+    def transform(obj):
+        try:
+            if isinstance(obj, (float, int)):
+                return int(obj * 10)
+            if obj.isnumeric():
+                return int(obj) * 10
+            if obj.replace(".", "").isnumeric():
+                return int(float(obj) * 10)
+            return None
+        except ValueError:
+            LOGGER.error("%s connector error, cannot convert input : %s", connector, obj)
+            raise
+
+
+class SizeToInteger(ValueTransformer):
+    """A value transformer for converting value to integer value
+    Example:
+        Input: '19261 bytes'
+        Output: 19261
+    """
+    @staticmethod
+    def transform(obj):
+        try:
+            if obj:
+                return int(str(obj).replace(" bytes", ""))
+            return None
+        except ValueError:
+            LOGGER.error("%s connector error, cannot convert input : %s", connector, obj)
+            raise
+
+
+class ToProtocolValue(ValueTransformer):
+    """A value transformer for converting value to protocol value
+    Example:
+        Input: 'tcp/3389' or 'unknown'
+        Output: [tcp] or [tcp]
+    """
+    @staticmethod
+    def transform(obj):
+        try:
+            if obj:
+                if '/' in obj:
+                    protocol = obj.split("/")[0]
+                    return [protocol]
+                if obj == 'unknown':
+                    return ['tcp']
+                return [obj.lower()]
```

Comment:
Should this lower() be applied to the return statements in the if statements as well?

Reply:
Updated as per the suggestion.